pyfolio/plotting.py

```diff
@@ -610,16 +629,19 @@ def plot_rolling_returns(
             lw=4, color='red', alpha=0.6,
             label='Live', ax=ax, **kwargs)
 
-        if cone_std is not None:
-            cone_df = timeseries.cone_rolling(
-                returns,
-                num_stdev=cone_std,
-                cone_fit_end_date=live_start_date)
-
-            cone_df_fit = cone_df[cone_df.index < live_start_date]
+        if cone_std is not None and not volatility_match:
+            # check to see if cone_std was passed as a single value and,
+            # if so, just convert to list automatically
+            if isinstance(cone_std, float):
+                cone_std_list = [cone_std]
+            else:
```

Comment:
Thanks for pointing that out (I actually didn't realize i could re-use a parameter variable like that.  Is this only possible because the parameter does not have risk of actually just being a pointer to an upstream variable? I'm always confused as to when Python treats an input parameter as merely a pointer to the upstream variable)